hack/install-odf.sh

```diff
@@ -20,6 +20,15 @@ fi
 
 "$OPERATOR_SDK" run bundle "$BUNDLE_IMG" --timeout=10m --security-context-config restricted -n "$INSTALL_NAMESPACE" --index-image "$CATALOG_DEPS_IMG"
 
+# Check for the presence of the CSVs in the cluster for up to 5 minutes,
+# Since 'oc wait' exits immediately if the resource is not found.
```

Comment:
probably because of non zero exit code from `oc wait` did you try with `--ignore-node-found`? if we go by dependency chain why not wait for the deps csv status to be successful? sleep is almost always troublesome.

Reply:
I will try this though i could not found anything helpful in the `oc wait help`

Reply:
```
┌─[nigoyalॐ fedora ~/code/go/src/github.com/red-hat-storage/odf-operator  bundle]
└──╼$ oc wait --timeout=10s --for condition=Available deployment catalog-operatora
Error from server (NotFound): deployments.apps "catalog-operatora" not found

┌─[nigoyalॐ fedora ~/code/go/src/github.com/red-hat-storage/odf-operator  bundle]
└──╼$ oc wait --timeout=10s --for condition=Available deployment catalog-operatora --ignore-not-found
error: unknown flag: --ignore-not-found
See 'oc wait --help' for usage.
```

I tried this, This flag is not available in wait.

Even if we check the deps csv status we have to check for its presence before checking the status. For checking its presence we have to wait and sleep.

Reply:
ack, resolve.